unime/src-tauri/tests/tests/get_state.rs

```diff
@@ -39,6 +39,34 @@ async fn test_get_state_create_new() {
     .await;
 }
 
+#[tokio::test]
+#[serial_test::serial]
+async fn test_locale_persistency_during_onboarding() {
+    setup_state_file();
+    setup_stronghold();
+
+    // Deserializing the AppStates and Actions from the accompanying json files.
```

Comment:
I don't think this comment adds more meaning to the code since it is a quite common test setup.

Reply:
see comment below